iptables/iptables.go

```diff
@@ -423,12 +423,22 @@ func existsRaw(table Table, chain string, rule ...string) bool {
 	return strings.Contains(string(existingRules), ruleString)
 }
 
+func filterOutput(output []byte, args ...string) []byte {
+	// ignore iptables' message about xtables lock
+	if strings.Contains(string(output), xLockWaitMsg) {
+		logrus.Warnf("xtables contention detected while running [%s]: %q", strings.Join(args, " "), string(output))
```

Comment:
I see what you meant regarding the warning, should we follow another approach for the warning, like if the operation takes more than x seconds to execute than we print it?

Reply:
Hrmm..  Yes, I could see that being a good compromise.  Gives us more control over the results in any case.  I'll write it up and update.